acm/registration/odh-core.application.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: odh-core
+  namespace: rhoai-edge-acm
+spec:
+  componentKinds:
+  - group: apps.open-cluster-management.io
+    kind: Subscription
+  descriptor: {}
+  selector:
+    matchExpressions:
+      - key: app
+        operator: In
+        values: 
+          - odh-core
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ggithubcom-opendatahub-io-ai-edge-ns
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: Channel
+metadata:
+  annotations:
+    apps.open-cluster-management.io/reconcile-rate: medium
+  name: ggithubcom-opendatahub-io-ai-edge
```

Comment:
If this channel being in a different namespace is expected to be re-used in other subscriptions, like when we've got an `odh-edge` one, then should it (and the ns definition above it) be repeated in the `odh-edge.yaml`, or should it be pulled out of this file into a common one?